bylaws.md

```diff
@@ -470,6 +470,14 @@ than the election of Cohort A directors. Officer Directors can also be
 removed subject to a vote of no confidence by the at-large Directors.
 In all other respects Officer Directors shall be as at-large Directors.
 
+**Section 5.5.2 Term Limits for Directors.** 
+No director shall serve more than six consecutive years. If a director
+has served for six years, that director shall become eligible again serve 
+on the Board of Directors when the directors are seated following the 
+election of the at-large directors in the following calendar year.
```

Comment:
While I would argue that `on the Board of Directors following the election of at-large directors in the following calendar year'  reads more clearly I presume the language about being seated stems from the fact that an election results != necessarily those same names being the ones who fulfill the at-large roles.  Is that correct?

Reply:
Updated to use something closer to your suggested phrasing.